client/app/components/parameter-settings.html

```diff
@@ -29,9 +29,14 @@ <h4 class="modal-title">{{ $ctrl.isNewParameter ? 'Add Parameter' : $ctrl.parame
           <option value="number">Number</option>
           <option value="enum">Dropdown List</option>
           <option value="query">Query Based Dropdown List</option>
+          <option disabled>--------------------------------------------------</option>
           <option value="date">Date</option>
           <option value="datetime-local">Date and Time</option>
           <option value="datetime-with-seconds">Date and Time (with seconds)</option>
+          <option disabled>--------------------------------------------------</option>
+          <option value="date-range">Date Range</option>
+          <option value="datetime-range-local">Date and Time Range</option>
```

Comment:
The `-local` suffix is the result of laziness on my part (we were using the parameter type as the `<input>` tag type). No need to pass it as legacy to new types :)

`datetime-range` is enough.